### PR TITLE
Puppet gcp simpleapp

### PR DESCRIPTION
**Pull request by nelsonjr**
Step-by-step on how to create a machine in GCP and setup Apache + PHP on it.

### CODE REVIEW

File: google/setup_puppet_and_apache_google-cloud-platform.md [hunk 1]
```diff
@@ -0,0 +1,348 @@
...
+[Developer Console][] or the 'gcloud' tool (a.k.a. [Google Cloud SDK][]).
+
+This tutorial is based on Cent OS 7, so make sure your machine is created using
+that operating system, or you'll need to change the commands to match your other
+OS.
+
+Using the gcloud tool:
+
+```
+gcloud compute instances create my-first-app \
```
Comment (va-rau-jo) [inline google/setup_puppet_and_apache_google-cloud-platform.md:30]: Add machine type.

Comment (nelsonjr) [reply google/setup_puppet_and_apache_google-cloud-platform.md:30]: Done.

Comment (nelsonjr) [reply google/setup_puppet_and_apache_google-cloud-platform.md:30]: Better? https://github.com/nelsonjr/puppet-tips/blob/463ee2d91f706a724a68f13e18b07631f2ffd635/google/setup_puppet_and_apache_google-cloud-platform.md#get-a-machine

File: google/setup_puppet_and_apache_google-cloud-platform.md [hunk 2]
```diff
@@ -0,0 +1,358 @@
...
+would be printed until they (or the community) update this module.
+
+To avoid having to see those warnings go by we'll turn warnings off. This should
+be a temporary action and you should turn it back on as soon as possible.
+
+To be cute we'll use Puppet to configure Puppet. I find it cool a tool can
+configure itself :)
+
+```puppet
+file_line { 'turn-warnings-off':
```
Comment (va-rau-jo) [inline google/setup_puppet_and_apache_google-cloud-platform.md:267]: Put in any file and apply it

File: google/setup_puppet_and_apache_google-cloud-platform.md [hunk 3]
```diff
@@ -0,0 +1,358 @@
...
+[root@my-first-app ~]# 
+```
+
+## Installing software
+
+### The "hard" way (automated, pero-no-mucho)
+
+You can tell Puppet to install software on your machine easily. You just declare
+a `package` block. For example, to install Apache (web server) on our Cent OS
+machine we crate a file, say `apache.pp`:
```
Comment (va-rau-jo) [inline google/setup_puppet_and_apache_google-cloud-platform.md]: typo :-1:

Comment (nelsonjr) [reply google/setup_puppet_and_apache_google-cloud-platform.md]: Done

**Pull request merged**